### PR TITLE
Add trace agent attribute to stackdriver protos to export

### PR DESCRIPTION
**Pull request by dazuma**
Fixes https://github.com/census-instrumentation/opencensus-ruby/issues/57

/cc @bogdandrutu

### CODE REVIEW

File: lib/opencensus/trace/exporters/stackdriver/converter.rb [hunk 1]
```diff
@@ -66,7 +81,8 @@ def convert_span obj
               end_time: convert_time(obj.end_time),
               attributes:
                 convert_attributes(obj.attributes,
-                                   obj.dropped_attributes_count),
+                                   obj.dropped_attributes_count,
+                                   include_agent_attribute: true),
```
Comment (chingor13) [inline lib/opencensus/trace/exporters/stackdriver/converter.rb:85]: Do you want this on every span or only on the root span?

File: lib/opencensus/trace/exporters/stackdriver/converter.rb [hunk 2]
```diff
@@ -37,6 +37,21 @@ class Converter
...
+          AGENT_KEY = "g.co/agent".freeze
+
+          ##
+          # @private
+          # Attribute value for Stackdriver trace agent
+          #
+          AGENT_VALUE =
+            OpenCensus::Trace::TruncatableString.new \
+              "opencensus-ruby [#{::OpenCensus::VERSION}] ruby-stackdriver-" \
+              "exporter [#{::OpenCensus::Stackdriver::VERSION}]"
```
Comment (chingor13) [inline lib/opencensus/trace/exporters/stackdriver/converter.rb:53]: @bogdandrutu Is this format with the core version and exporter version correct?

If so, I'll do this for the php exporter as well.

File: lib/opencensus/trace/exporters/stackdriver/converter.rb (hunk 1 above)
Comment (dazuma) [reply lib/opencensus/trace/exporters/stackdriver/converter.rb:85]: @chingor13 Good question. Given that different parts of the span tree may be reported by different apps, I'm guessing that we'd want this on every span. What do you guys do for PHP?

@bogdandrutu Any comment on this?

Comment (chingor13) [reply lib/opencensus/trace/exporters/stackdriver/converter.rb:85]: For PHP, we're only putting the agent label on the root span, but we can put it on all of them.

Comment (bogdandrutu) [reply lib/opencensus/trace/exporters/stackdriver/converter.rb:85]: I think it is better on all of them.

**Review by chingor13 — APPROVED** [review]

**Pull request merged**